contracts/mainnet/connectors/liquity/main.sol

```diff
@@ -50,23 +29,23 @@ abstract contract LiquityResolver is Events, Helpers {
      * @param borrowAmount The amount of LUSD to borrow
      * @param upperHint Address of the Trove near the upper bound of where the user's Trove should now sit in the ordered Trove list
      * @param lowerHint Address of the Trove near the lower bound of where the user's Trove should now sit in the ordered Trove list
-     * @param getId Optional storage slot to retrieve ETH from
-     * @param setId Optional storage slot to store the LUSD borrowed against
+     * @param getIds Optional (default: 0) Optional storage slot to get deposit & borrow amounts stored using other spells
+     * @param setIds Optional (default: 0) Optional storage slot to set deposit & borrow amounts to be used in future spells
     */
     function open(
         uint depositAmount,
         uint maxFeePercentage,
         uint borrowAmount,
         address upperHint,
         address lowerHint,
-        uint getId,
-        uint setId
+        uint[] getIds,
+        uint[] setIds
     ) external payable returns (string memory _eventName, bytes memory _eventParam) {
-        if (getId != 0 && depositAmount != 0) {
-            revert("open(): Cannot supply a depositAmount if a non-zero getId is supplied");
-        }
```

Comment:
Is there a problem with this type of check?

Reply:
I think it doesn't matter much. `getId` will overwrite the `depositAmount` if it's anything other than 0. So the checks aren't important here but the checks were not wrong.

Reply:
Ah okay, I was trying to protect developers from incorrectly using the API since there’s no indicator that the value is ignored when a getId is supplied. This way there less likely to encounter bugs.

Reply:
Yeah! I understood it. Although, the assets will either be in user's Liquity position or in DSA so it's alright. Anyway, we might think of doing these minor updates if makes sense in all our connectors at once but let's go like this for now :)